preset/preset_builtin.json

```diff
@@ -8797,7 +8797,7 @@
                     "SubtitleTrackSelectionBehavior": "none",
                     "VideoAvgBitrate": 6500,
                     "VideoColorMatrixCodeOverride": 0,
-                    "VideoEncoder": "qsv_av1_10bit",
+                    "VideoEncoder": "qsv_av1",
```

Comment:
This was deliberately 10bit

Reply:
Do I need to delete this commit from PR?

Reply:
Yes please. 😎

Reply:
Done